docs/content/configuration/creating_volumegroup.md

```diff
@@ -0,0 +1,43 @@
+# Creating a VolumeGroup
+
+Create a VolumeGroup YAML file to specify a volume group key, for creating PersistentVolumeClaim (PVC) groups.
+
+VolumeGroupClass needs to be present before a VolumeGroup can be created. For more information, see [Creating a VolumeGroupClass](creating_volumegroupclass.md).
+
+Before creating a volume group, be sure to follow all of the volume group configurations, found in [Compatibility and requirements](../installation/install_compatibility_requirements.md).
+
+1.  Create a Volume Group using the `demo-volumegroup.yaml`.
+
+    **Note:** Use the `key` value is used in the relevant target PersistentVolumeClaim (PVC).
```

Comment:
used or will be used*, it depends on whether he creates the PVCs first and and then the volumegroup,
or vice versa

Reply:
updated